include/unwind_output.hpp

```diff
@@ -24,8 +25,10 @@ struct UnwindOutput {
   void clear() {
     locs.clear();
     is_incomplete = true;
+    container_id = "unknown";
```

Comment:
Ideally we'd use a default value that conforms to the right part of the Datadog BE.  Ultimately, I think the huge incremental value is going to be relating the sub-profiles to different things we harvest in the product.

That said, the immediate consumer is the profiling BE, and whenever we have a NULL context I think we have a challenge with upgrading it into a distinct type (for filtering/discoverability), so this is good.

Reply:
Yeah, we might need to revisit this. I'm making it at least more transparent in the next commit.